packages/config/config/devices/0x024e/WIN-GDC-02.json

```diff
@@ -0,0 +1,323 @@
+			"defaultValue": 10
+		},
+		"4": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 Auto Off Delay",
+			"description": "Relay will turn off automatically after time specified in this parameter. After receiving command On Relay 1 will be switched Off automatically with predefined delay."
+		},
+		"5": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 On Delay",
+			"description": "Relay 1 will be switched On with predefined delay."
+		},
+		"6": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 Off Delay",
+			"description": "Relay 1 will be switched Off with predefined delay."
+		},
+		"7": {
+			"$import": "../templates/master_template.json#state_after_power_failure_prev_off",
+			"label": "Relay 1 Restore After Power Loss",
+			"description": "Restore to the last known state for relay 1 after power was interrupted."
+		},
+		"8": {
+			"label": "Broadband Command Action",
+			"description": "Broadband command turn all On/ turn all Off settings.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Accept broadband command turn all On/ turn all Off",
+					"value": 0
+				},
+				{
+					"label": "Ignore broadband command turn all On/ turn all Off",
+					"value": 1
+				},
+				{
+					"label": "Ignore broadband command turn all On, accept broadband command turn all Off",
+					"value": 2
+				},
+				{
+					"label": "Accept broadband command turn all On, ignore broadband command turn all Off",
+					"value": 3
+				}
+			]
+		},
+		"9": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Hide Relay",
+			"description": "Hide relay 2 in controller interface"
+		},
+		"10": {
+			"$import": "templates/zconnect_template.json#relay_mode_selection",
+			"label": "Relay 2 Mode"
+		},
+		"11": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 2 Auto Off Delay",
+			"description": "Relay will turn off automatically after time specified in this parameter. After receiving command On Relay 2 will be switched Off automatically with predefined delay."
+		},
+		"12": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 2 On Delay",
+			"description": "Relay 2 will be switched On with predefined delay."
+		},
+		"13": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 2 Off Delay",
+			"description": "Relay 2 will be switched Off with predefined delay."
+		},
+		"14": {
+			"$import": "../templates/master_template.json#state_after_power_failure_prev_off",
+			"label": "Relay 2 Restore After Power Loss",
+			"description": "Restore to the last known state for relay 2 after power was interrupted."
+		},
+		"15": {
+			"label": "Relay 1&2 Association",
```

Comment:
Let's make this "Control Relay 2 with Relay 1" or something. We want as many of these to be disasble/enable as we can as those will eventually be made switches by some of the UIs.

Make the options disable/enable. There is a base tempalte for disable/enable in the master template.

Reply:
The description is a bit weird. Do they work together as a unit? Or does only On on R1 affect R2? Have tidied it up as suggested, but left the meaning pretty much consistent with the documentation.

Reply:
Let's just say "Control Relays 1 & 2 Together" or something than. I'd like to make it enable/disable.

Reply:
Unfortunately I don't have anything wired to R2 it to test it with. Have updated as per your suggestion.